frontend/src/v5/validation/containerAndFerderationSchemes/federationSchemes.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
```

Comment:
The directory these files are in has a typo

Reply:
fixed